TWLight/emails/tests.py

```diff
@@ -240,6 +243,17 @@ def test_approval_calls_email_function(self, mock_email):
         app.save()
         self.assertTrue(mock_email.called)
 
+    def test_approval_does_not_call_email_for_applications_with_access_codes_partner(
+        self,
+    ):
+        app_with_access_codes_partner = ApplicationWithAccessCodePartnerFactory(
+            status=Application.PENDING
+        )
+        app_with_access_codes_partner.status = Application.APPROVED
+        app_with_access_codes_partner.save()
+        orig_outbox = len(mail.outbox)
+        self.assertEqual(len(mail.outbox), orig_outbox)
```

Comment:
Instead of this assertion, can you change it to `self.assertEqual(len(mail.outbox), 0)`? I want it to be explicit in the assertion that no emails are being sent.